### ISSUE TITLE
Tokenizer is corrupting images

### ISSUE THREAD

**Issue by dubiousgit** [MrPrimate/tokenizer#175]
Not really sure what to do about this one as it was working fine, until it wasn't...

![image](https://github.com/user-attachments/assets/9d834ec2-c4d6-4d66-b56d-8789eea2738b)

Whenever I 'tokenize' an image now, it corrupts as seen in the screenshot.

This doesn't happen if you manually fiddle with the token (the top one in the screenshot) and create it without the automation.

**Comment by MrPrimate** [comment]
I cannot replicate this, please provide:
- Exact Foundry and system versions you are using.
- Have you tried with just Tokenizer module enabled?
- Detailed step by step to replicate

**Comment by dubiousgit** [comment]
> I cannot replicate this, please provide:
> 
>     * Exact Foundry and system versions you are using.

The Forge,
D&D5e 3.3.1,
Foundry v12 Stable (build 331)


>     * Have you tried with just Tokenizer module enabled?

Yes, other than 'The Forge' which I can't disable, still happens:

![image](https://github.com/user-attachments/assets/0d894e5f-cd90-4638-b167-94145eb17e89)

>     * Detailed step by step to replicate

Import monsters with D&D importer.

Select 'Tokenize' on import, or do it post-import by right clicking in the compendium.

**Comment by dubiousgit** [comment]
I should note that I also tried this on a brand new world, same thing happened.

**Comment by dubiousgit** [comment]
> If you import with out the auto tokenize selected, does it create a token?

Yes, and the token looks fine (in that it's the default, square one).

I can also manually add a token fine, though obviously that would be labour intensive.

Only when importing and ticking that box, or right clicking on the compendium and 'auto tokenizing' does it cause the corrupted image.

**Comment by dubiousgit** [comment]
I *do* make use of the multi-world 'game manager' feature on Forge, maybe this is causing some sort of confusion with the assets? They share an assets blob as far as I can tell.

**Comment by MrPrimate** [comment]
I have logged into my Forge account and can't replicate this - you've not run out of space or anything in your Assets folder?

What are the paths set to for tokenizer and ddb importer for saving images?

**Comment by dubiousgit** [comment]
> I have logged into my Forge account and can't replicate this - you've not run out of space or anything in your Assets folder?
> 
> What are the paths set to for tokenizer and ddb importer for saving images?

It's a new account, still have several gigs of space.

Paths are the default, (see assets screenshot above).

Is your Forge set to the 'Game Manager' mode?

**Comment by dubiousgit** [comment]
![image](https://github.com/user-attachments/assets/db509885-542b-4c66-9fd5-edbb98a828e3)

**Comment by dubiousgit** [comment]
![image](https://github.com/user-attachments/assets/4d64f416-1b63-442f-b7c3-cc6bf59a2fe1)

**Comment by dubiousgit** [comment]
I also tried saving as PNG in case it was something weird with webp, but the same thing happens.

**Comment by MrPrimate** [comment]
> > I have logged into my Forge account and can't replicate this - you've not run out of space or anything in your Assets folder?
> > 
> > What are the paths set to for tokenizer and ddb importer for saving images?
> 
> It's a new account, still have several gigs of space.
> 
> Paths are the default, (see assets screenshot above).
> 
> Is your Forge set to the 'Game Manager' mode?
> 

I want to see the paths in the settings, as forge has its own unique way of specifying paths.

**Comment by dubiousgit** [comment]
@MrPrimate these?

![image](https://github.com/user-attachments/assets/ae2bc25d-e062-4542-b12d-69bde840bf74)

**Comment by MrPrimate** [comment]
Can you try using Chrome for me rather than Firefox?

**Comment by dubiousgit** [comment]
@MrPrimate I don't have Chrome specifically installed but I can confirm it worked in a Chromium-based browser!
